docs/guide/clean-up.md

```diff
@@ -45,9 +45,7 @@ import { CounterDefaultState, ICounterState } from './counter/state';
 
 ...
 
-  counter: {
     ...CounterDefaultState(),
-  },
```

Comment:
hey, if you only remove `...CounterDefaultState(),` you would still have `counter:{}` in the state. This should also be removed. Are you sure the app wont work by removing the whole `counter` state entry?